src/test/java/uk/gov/register/functional/ApplicationTest.java

```diff
@@ -82,10 +83,27 @@ public void appSupportsXssProtection() throws Exception {
     public void app404PageHasXhtmlLangAttributes() throws Exception {
         Response response = register.getRequest(address, "/missing_page_to_force_a_404");
 
+        assertThat(response.getStatus(), equalTo(404));
         Document doc = Jsoup.parse(response.readEntity(String.class));
         Elements htmlElement = doc.select("html");
         assertThat(htmlElement.size(), equalTo(1));
         assertThat(htmlElement.first().attr("lang"), equalTo("en"));
         assertThat(htmlElement.first().attr("xml:lang"), equalTo("en"));
     }
+
+    @Test
+    public void returns405ForNotAllowedMethod() {
+        Response response = register.getRequest(address, "/load-rsf");
+
+        assertThat(response.getStatus(), equalTo(405));
+    }
+
+    @Test
+    public void returns400BadRequest() {
+        Response response = register.authenticatedTarget(address).path("/load-rsf").request()
```

Comment:
I knew you were going to pick on that :D heh there is a test already `DataUploadTest` which uses
`register.mintLines` to cause that exception. My intention here was to be more explicit when it comes to what is actually happening and exposing how I can achieve 400 without investigating methods like `register.loadRsf("nope")`. But fair enough it doesn't look less informative when changed to `loadRsf`. Will change that.